common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -291,6 +303,8 @@ func (m *cassandraMetadataPersistenceV2) UpdateDomain(request *p.InternalUpdateD
 		request.ConfigVersion,
 		request.FailoverVersion,
 		request.FailoverNotificationVersion,
+		failoverConfig.StartTime,
```

Comment:
Is there validation we need to do on StartTime and and Timeout. A few thoughts 

- They should either both be set or neither should be set any other combination is invalid?
- Do we want protections on how long this timeout should be?

Reply:
Yes. I changed them to failover end time and only update one field now. The protection logic will be in business logic.